implementation/jobberknoll/core/deno.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "@jobberknoll/core",
-  "version": "0.1.0",
```

Comment:
Consolidating separate versions for subpackages in the workspace to a single workspace version.